### ISSUE TITLE
calico-kube-controllers is in CreateContainerConfigError when allowPrivilegedContainer is false

### ISSUE THREAD

**Issue by ialidzhikov** [gardener/gardener#1119]
1. Create Shoot with `.spec.kubernetes.allowPrivilegedContainers: false`.
2. Ensure that
```
$ k get po -n kube-system -l k8s-app=calico-kube-controllers
calico-kube-controllers-65fdcc67d4-jmh5s                          0/1     CreateContainerConfigError   0          7m41s

$ k get po -n kube-system -l k8s-app=calico-kube-controllers -o yaml
    state:
      waiting:
        message: container has runAsNonRoot and image will run as root
        reason: CreateContainerConfigError
```

**Comment by rfranzke** [comment]
Nice finding. Do you want to take that over @ialidzhikov and add the missing PSPs? @mvladev can probably consult and help!

**Comment by mvladev** [comment]
I've fixed it in #1120 , but I'll file a PR upstream to fix it in calico, so we don't need this PSP anymore.
